package.json

```diff
@@ -68,58 +68,58 @@
   },
   "homepage": "https://github.com/bbc/simorgh#readme",
   "dependencies": {
-    "@bbc/gel-foundations": "6.0.0",
+    "@bbc/gel-foundations": "6.1.1",
     "@bbc/moment-timezone-include": "1.1.4",
-    "@bbc/psammead-amp-geo": "1.2.0",
-    "@bbc/psammead-assets": "3.1.2",
-    "@bbc/psammead-brand": "7.0.17",
-    "@bbc/psammead-bulleted-list": "3.0.5",
-    "@bbc/psammead-bulletin": "5.0.16",
-    "@bbc/psammead-byline": "3.0.5",
-    "@bbc/psammead-calendars": "2.0.19",
-    "@bbc/psammead-caption": "4.1.2",
-    "@bbc/psammead-consent-banner": "4.0.6",
-    "@bbc/psammead-copyright": "3.0.6",
-    "@bbc/psammead-detokeniser": "1.0.0",
-    "@bbc/psammead-embed-error": "3.0.8",
-    "@bbc/psammead-episode-list": "1.0.5",
-    "@bbc/psammead-figure": "2.0.1",
-    "@bbc/psammead-grid": "3.0.9",
-    "@bbc/psammead-heading-index": "3.0.6",
-    "@bbc/psammead-headings": "5.0.6",
-    "@bbc/psammead-image": "2.0.1",
-    "@bbc/psammead-image-placeholder": "3.0.8",
-    "@bbc/psammead-inline-link": "3.0.5",
-    "@bbc/psammead-live-label": "2.0.7",
-    "@bbc/psammead-locales": "5.0.0",
-    "@bbc/psammead-media-indicator": "6.0.8",
-    "@bbc/psammead-media-player": "5.0.15",
-    "@bbc/psammead-most-read": "6.0.14",
-    "@bbc/psammead-navigation": "9.0.1",
-    "@bbc/psammead-paragraph": "4.0.6",
-    "@bbc/psammead-podcast-promo": "0.1.0-alpha.24",
-    "@bbc/psammead-radio-schedule": "5.1.7",
-    "@bbc/psammead-rich-text-transforms": "2.0.2",
-    "@bbc/psammead-script-link": "3.0.7",
-    "@bbc/psammead-section-label": "7.0.7",
-    "@bbc/psammead-sitewide-links": "6.0.6",
-    "@bbc/psammead-social-embed": "3.1.3",
-    "@bbc/psammead-story-promo": "8.0.8",
-    "@bbc/psammead-story-promo-list": "6.0.6",
-    "@bbc/psammead-styles": "7.0.3",
-    "@bbc/psammead-timestamp-container": "5.0.16",
-    "@bbc/psammead-useful-links": "3.0.7",
-    "@bbc/psammead-visually-hidden-text": "2.0.1",
+    "@bbc/psammead-amp-geo": "1.2.2",
+    "@bbc/psammead-assets": "3.1.5",
+    "@bbc/psammead-brand": "7.0.30",
+    "@bbc/psammead-bulleted-list": "3.0.12",
+    "@bbc/psammead-bulletin": "5.0.30",
+    "@bbc/psammead-byline": "3.0.13",
+    "@bbc/psammead-calendars": "2.0.23",
+    "@bbc/psammead-caption": "4.1.8",
+    "@bbc/psammead-consent-banner": "5.3.2",
```

Comment:
Major bump here - checking w/ engage - https://github.com/bbc/psammead/blob/latest/packages/components/psammead-consent-banner/CHANGELOG.md

Reply:
Can't update this 😦 - this PR is blocked for the minute - can't update `psammead-consent-banner` to `5.x` as it has a11y issues that Engage are working on; can't update to emotion 11 with this component on `4.x` as the emotion 10 + emotion 11 combination causes style tags in the body of the document, thereby failing AMP validation

https://github.com/bbc/simorgh/pull/8827